packages/volto/news/6545.feature

```diff
@@ -0,0 +1,6 @@
+Improved undo control panel. @giuliaghisini
+- Now it displays a loader while waiting for transactions from Plone.
+- When you want to delete some transactions, a confirmation popup asks the user to confirm the undo action for the selected items.
+- Changed columns' width to better display path.
+- Added some translations.
+- Added style to accessibility focus on checkbox.
```

Comment:
@giuliaghisini this entry is no longer valid after this revert commit of yours
https://github.com/plone/volto/pull/6545/commits/650f5b4cf4692e908fb396f359a392a6c492fc6c#diff-3a5f2698b3ccbd44cd87e2f818c58afbd7d9981a3c90afbaf56a80cf2c93f9c2

Reply:
yes, i've updated the description